app/jobs/notification/reopen_snoozed_notifications_job.rb

```diff
@@ -0,0 +1,9 @@
+class Notification::ReopenSnoozedNotificationsJob < ApplicationJob
+  queue_as :low
+
+  def perform
+    Notification.where.not(snoozed_until: nil).where(snoozed_until: 3.days.ago..Time.current).find_each(batch_size: 100) do |notification|
+      notification.update(snoozed_until: nil)
+    end
```

Comment:
Can we instead do this in a single query?

```rb
Notification.where.not(snoozed_until: nil)
                .where(snoozed_until: 3.days.ago..Time.current)
                .update_all(snoozed_until: nil)
```

In case we also want to change the `updated_at` we can add `updated_at: Time.current` in the `update_all` query

CC: @sojan-official WDYT?

Reply:
@scmmishra Makes sense. I have updated the PR.